src-tauri/src/main.rs

```diff
@@ -628,3 +631,30 @@ fn launch_northstar_steam(
 
     retval
 }
+
+#[tauri::command]
+async fn install_northstar_proton_wrapper() -> Result<(), String> {
```

Comment:
Maybe add a doc comment for wrapper functions as well. So for example

```suggestion
/// On Linux attempts to install NorthstarProton
/// On Windows simply returns an error message
#[tauri::command]
async fn install_northstar_proton_wrapper() -> Result<(), String> {
```